LoadIndicator: Update demo description (#2058)
DevExpress/devextreme-demos: JSDemos/Demos/LoadIndicator/Overview/description.md

> The LoadIndicator component notifies users that a process is in progress. In this demo, you can see how to initialize and configure the component as well as use it within other components.

Reviewer: We don't use "as well as" because it's basically the same as "and".
I also suggest replacing the word "use" with "integrate" in this particular case. It's not a mistake, just avoiding overused words.

```suggestion
The LoadIndicator component notifies users that a process is in progress. In this demo, you can see how to initialize and configure the component and integrate it into other components.
```

## Specify Dimensions

> LoadIndicator allows you to specify the indicator's dimensions. Set the [height](/Documentation/ApiReference/UI_Components/dxLoadIndicator/Configuration/#height) and [width](/Documentation/ApiReference/UI_Components/dxLoadIndicator/Configuration/#width) properties to the size you need. In this demo, the first three components have different sizes.

Reviewer: LoadIndicator is the indicator itself.

```suggestion
LoadIndicator allows you to specify its dimensions. Set the [height](/Documentation/ApiReference/UI_Components/dxLoadIndicator/Configuration/#height) and [width](/Documentation/ApiReference/UI_Components/dxLoadIndicator/Configuration/#width) properties to the size you need. In this demo, the first three components have different sizes.
```

## Apply a Custom Indicator

> If you want to apply a custom indicator, assign the path to a new image to the [indicatorSrc](/Documentation/ApiReference/UI_Components/dxLoadIndicator/Configuration/#indicatorSrc) property. In this demo, the fourth LoadIndicator displays the custom horizontal indicator.

Reviewer: Too many entries of the word "indicator" in this description. It's understandable, because of the subject matter, but still we should try to substitute frequently occuring words if we can.

```suggestion
If you want to apply a custom indicator, assign the path to a new image to the [indicatorSrc](/Documentation/ApiReference/UI_Components/dxLoadIndicator/Configuration/#indicatorSrc) property. In this demo, the fourth LoadIndicator displays a custom animated GIF.
```

> ## Use within Other Components

Reviewer: ```suggestion
## Integrate LoadIndicator into Other Components
```
